include/radio_types.h

```diff
@@ -2124,9 +2138,43 @@ typedef struct radio_cell_info_1_5 {
         RadioCellInfoNr_1_5 nr RADIO_ALIGNED(8);
         RadioCellInfoCdma_1_2 cdma RADIO_ALIGNED(8);
     } info RADIO_ALIGNED(8);
-} RADIO_ALIGNED(8) RadioCellInfo_1_5; /* Since 1.5.0 */
+} RADIO_ALIGNED(8) RadioCellInfoRatSpecificInfo_1_5; /* Since 1.5.4 */
+G_STATIC_ASSERT(sizeof(RadioCellInfoRatSpecificInfo_1_5) == 192);
+
+typedef struct radio_cell_info_1_5 {
+    guint8 registered RADIO_ALIGNED(1);
+    gint32 timeStampType RADIO_ALIGNED(4);
+    guint64 timeStamp RADIO_ALIGNED(8);
+    RADIO_CELL_CONNECTION_STATUS connectionStatus RADIO_ALIGNED(4);
+    RadioCellInfoRatSpecificInfo_1_5 ratSpecificInfo RADIO_ALIGNED(8);
+} RADIO_ALIGNED(8) RadioCellInfo_1_5; /* Since 1.5.4 */
 G_STATIC_ASSERT(sizeof(RadioCellInfo_1_5) == 216);
 
+typedef struct radio_physical_channel_config {
+    RADIO_CELL_CONNECTION_STATUS connectionStatus RADIO_ALIGNED(4);
+    gint32 cellBandwidthDownlink RADIO_ALIGNED(4);
+} RADIO_ALIGNED(4) RadioPhysicalChannelConfig; /* Since 1.5.4 */
+G_STATIC_ASSERT(sizeof(RadioPhysicalChannelConfig) == 8);
+
+typedef struct radio_frequency_info {
+    guint8 frequencyInfoType RADIO_ALIGNED(4); /* RADIO_FREQUENCY_INFO_TYPE */
```

Comment:
Did you mean `RADIO_FREQUENCY_TYPE`? Or else rename RADIO_FREQUENCY_TYPE enum into RADIO_FREQUENCY_INFO_TYPE. And this field is `RADIO_ALIGNED(1)`:
```c
hidl_discriminator hidl_d __attribute__ ((aligned(1))) ;
```

Reply:
Fixed